Gemfile.lock

```diff
@@ -270,7 +270,7 @@ DEPENDENCIES
   rouge
 
 RUBY VERSION
-   ruby 2.6.3p62
+   ruby 2.7.1p83
```

Comment:
Can the Gemfile.lock file be removed from git?

Reply:
Not sure... so I'm going to leave it for now.